src/number-formatting/format.js

```diff
@@ -12,6 +12,37 @@ function formatNegativeNumber(str, options) {
 }
 
 /**
+ * converts a number to a decimal string if it is on scientific notation
+ * (method taken from https://gist.github.com/jiggzson/b5f489af9ad931e3d186)
+ * @param number
+ */
+function convertScientificToDecimal(number) {
+    // if the number is in scientific notation remove it
+    if(/\d+\.?\d*e[\+\-]*\d+/i.test(number)) {
+        var zero = '0';
+        var parts = String(number).toLowerCase().split('e');
+        var exponential = parts.pop();
```

Comment:
here you use parts.pop and below parts[0] - maybe parts[0] and parts[1] might be easier to follow?